app/channel_category.go

```diff
@@ -51,10 +64,13 @@ func (a *App) waitForSidebarCategories(userId, teamId string) (*model.OrderedSid
 		time.Sleep(100 * time.Millisecond)
 
 		categories, err := a.Srv().Store.Channel().GetSidebarCategories(userId, teamId)
+		if err != nil {
+			return nil, err
+		}
 
-		if err != nil || len(categories.Categories) > 0 {
+		if len(categories.Categories) > 0 {
```

Comment:
This actually changes the logic. Can we keep it the same?

Reply:
ok